src/openforms/submissions/migrations/0006_set_legacy_true.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.2.11 on 2024-06-26 14:26
+
+from django.db import migrations
+
+from django.db.migrations.state import StateApps
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+
+def set_legacy_true(apps: StateApps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    """Alter existing ``TemporaryFileUpload`` instances to have ``legacy`` set to ``True`` by default."""
+
+    TemporaryFileUpload = apps.get_model("submissions", "TemporaryFileUpload")
+
+    for file_upload in TemporaryFileUpload.objects.iterator():
+        file_upload.legacy = True
+        file_upload.save()
```

Comment:
```suggestion
    TemporaryFileUpload = apps.get_model("submissions", "TemporaryFileUpload")

    TemporaryFileUpload.objects.bulk_update(legacy=True)
```

Reply:
Assuming you meant `.update(legacy=True)`